Libplanet/Action/AccountStateDeltaImpl.cs

```diff
@@ -169,6 +176,17 @@ public virtual FungibleAssetValue GetTotalSupply(Currency currency)
             return TotalSupplyGetter(currency);
         }
 
+        [Pure]
+        public long UsedGas(Address address) =>
```

Comment:
Why do we separate gas usage per address?

Reply:
If do not separate gas usage, reset the previous gas usage end step of evaluating the transaction.
```
    for(action in tx):
        nextState = FeeCollect(action, nextState);
        nextState = Evaluate(action, nextState);
        nextState = RefundFee(action, nextState);
        nextState = nextState.ResetGas();
```

But separate by address, reset gas is no more needed.

```
    for(action in tx):
        nextState = FeeCollect(action, nextState);
        nextState = Evaluate(action, nextState);
        nextState = RefundFee(action, nextState);
```